README.md

```diff
@@ -182,70 +182,70 @@ MIGRATION: Upgrade to latest node.js version.
+The following changes are only relavant for plugin authors:
 
 ## Runtime Modules
 
-A large part of runtime code was moved into so called "runtime modules". These special modules add runtime code. They can be added in any chunk, but are currently always added to the runtime chunk. "Runtime requirements" control which runtime modules are added to the bundle. This ensures that only runtime code that is used is added to the bundle. In future runtime modules could also added to on-demand-loaded chunk to load runtime code when needed.
+A large part of the runtime code was moved into the so called "runtime modules". These special modules are in-charge of adding runtime code. They can be added in to any chunk, but are currently always added to the runtime chunk. "Runtime Requirements" control which runtime modules are added to the bundle. This ensures that only runtime code that is used is added to the bundle. In the future, runtime modules could also added to an on-demand-loaded chunk, to load runtime code when needed.
 
-The core runtime is now very small and only includes the `__webpack_require__` function, module factories and module instance cache. In future an alternative code runtime can be used to avoid wrapping the bundle in a IIFE and allow ESM style exports. This will allow ESM as target.
+The core runtime is now very small and only includes the `__webpack_require__` function, module factories and the module instance cache. In the future, an alternative code runtime can be used to avoid wrapping the bundle in a IIFE and allow ESM style exports. This will allow ESM as target.
 
-As some responsibilities from Main and ChunkTemplate were removed, also are some hooks.
+As some responsibilities from Main and ChunkTemplate were removed, several associated hooks were removed respectively.
 
-MIGRATION: When injection runtime code into the webpack runtime in a plugin, consider using RuntimeModules instead.
+MIGRATION: If you are injecting runtime code into the webpack runtime in a plugin, consider using RuntimeModules instead.
 
 ## Serialization
 
-A serialization mechanism was added to allow serialization of complex objects in webpack. It has a opt-in semantic, so classes that should be serialized need to be explicitly flagged (and serialization implemented). This has been done for most Modules, all Dependencies and some Errors.
+A serialization mechanism was added to allow serialization of complex objects in webpack. It has an opt-in semantic, so classes that should be serialized need to be explicitly flagged (and their serialization implemented). This has been done for most Modules, all Dependencies and some Errors.
 
-MIGRATION: When using custom Modules or Dependencies make sure to make them serializable to benefit from persistent caching.
+MIGRATION: When using custom Modules or Dependencies, make sure to provide them with serialization implemented, in order to benefit from persistent caching.
```

Comment:
```suggestion
MIGRATION: When using custom Modules or Dependencies, it is recommended to make them serializable to get the benefit from persistent caching.
```

Reply:
get the benefit is incorrect, it should either be `benefit from` or `get the benefit of`
```suggestion
MIGRATION: When using custom Modules or Dependencies, it is recommended to make them serializable to benefit from persistent caching.
```